### DISCUSSION TITLE
Workflow for categories AND subcategories

### DISCUSSION THREAD

**Discussion by hytsch** [joomla/joomla-cms#47011] [3. Approved Features]
### Does your feature request relate to a problem? Please describe it.

Inheritance plays an important role in the way Joomla works. Not only in the ACL, but also, for example, in the Custom Fields: assigning a field to a category automatically causes the field to be available in the subcategories as well.
Unfortunately, the functional concept for the workflow does not seem to have taken this into account: Each category requires a specification, otherwise the default workflow takes effect.

### Describe the solution you would like to have
We store workflow X in a category, and this automatically takes effect in all subcategories as well

# Approval Information
Approved: 18.03.2026
Minimum Target Version: 6.2

**Comment by HLeithner** [comment]
**The Joomla Maintainer Team decided at the 18.03.2025 meeting to accept this feature.**
